packages/arb-bridge-eth/contracts/rollup/facets/RollupUser.sol

```diff
@@ -214,7 +213,7 @@ abstract contract AbsRollupUserFacet is RollupBase, IRollupUser {
             CreateNodeDataFrame({
                 arbGasSpeedLimitPerBlock: arbGasSpeedLimitPerBlock,
                 confirmPeriodBlocks: confirmPeriodBlocks,
-                prevNode: latestStakedNode(msg.sender),
+                prevNode: latestStakedNode(msg.sender), // Ensure staker is staked on the previous node
```

Comment:
I don't think this comment is actually adding clarity.

Reply:
Mmm, i found this hard to find on my first pass. i.e., this is a thing that's important to enforce, and I want it to be clear (somehow) where in the code it is that we enforce it. Maybe there's a better/clearer way?